data and test cases for articles

gopalj opened the pull request:
Adding new endpoints and JSON data for articles scenario, it contains 1000 articles data.
with that, we can implement the below use cases
1. Using GET API -> Fetch articles for the given author and display only the article title field (List<String>), if the title is null use story_title
2. Using TWO GET API  -> Display the author details(about field from articles_users table) and the same author article title, if the title is null user story_title (from the articles table)

gopalj commented on package.json:
```diff
@@ -5,6 +5,7 @@
   "main": "app.js",
   "scripts": {
     "start": "node app.js",
+    "prepare-data": "node compile_data.js",
```
@layog - fixed it, please check.

layog replied:
Can you re-request review after updating the complete PR based on all the questions above?

gopalj replied:
sure, will do that.

gopalj replied:
@layog  - fixed the above concerns, please check.

kamaljoshi commented:
@gopalj When are you planning to add tests for the `article_users` endpoint. Also, consider removing some of the fields from the responses which won't be used(not necessary).

Also, do you have more use cases in mind then the ones you have mentioned?

gopalj commented:
Hi @kamaljoshi - I have added test cases for the article_users endpoint as well, also removed the unwanted fields from both the tables.

Please check it again.

gopalj commented:
@layog @kamaljoshi - Can you please check it once?

kamaljoshi commented on resources/article_users.json:
```diff
@@ -0,0 +1,93 @@
+  {
+    "id": 5,
+    "username": "olalonde",
+    "about": "olalonde@gmail.com<p>http:&#x2F;&#x2F;www.github.com&#x2F;olalonde<p>CTO&#x2F;Co-Founder @ https:&#x2F;&#x2F;binded.com",
```
Hey, can you use proper ascii '/'
